The code is encoding host compiler parameters into target builds. Avoid
this for our target builds (patch is target specific, not native)

Upstream-Status: Inappropriate [Cross compile hack]
RP 2020/2/18
Signed-off-by: Richard Purdie <richard.purdie@linuxfoundation.org>

Index: perl-5.30.1/cpan/Encode/bin/enc2xs
===================================================================
--- perl-5.30.1.orig/cpan/Encode/bin/enc2xs
+++ perl-5.30.1/cpan/Encode/bin/enc2xs
@@ -195,7 +195,7 @@ sub compiler_info {
     # above becomes false.
     my $sized  = $declaration && !($compat && !$pedantic);
 
-    return ($cpp, $static, $sized);
+    return (0, 1, 1);
 }
 
 
